tests/test_parsl_flux_mpi_hello.py

```diff
@@ -85,6 +85,10 @@ def test_flux_resource_list(load_config):
                       stderr=('parsl_flux_resource_list.err', 'w'),
                       env=load_config["environment"]).result()
     assert r == 0
+
+    pattern = re.compile(r'.*', re.DOTALL)
+    with open("parsl_flux_resource_list.out", "r") as f:
+        assert pattern.match(f.read())
```

Comment:
Then I can simply match my regex pattern to the contents of `parsl_flux_resource_list.out`